src/main/java/org/example/homework3/ArithmeticOperations.java

```diff
@@ -0,0 +1,20 @@
+package org.example.homework3;
+
+public class ArithmeticOperations {
+    public static void main(String[] args) {
+        int number_1 = 4;
```

Comment:
```suggestion
        int number1 = 4;
```
don't use underscore in variables. only camelcase